cmd/operator-sdk/add/controller.go

```diff
@@ -32,11 +32,17 @@ func newAddControllerCmd() *cobra.Command {
 		Use:   "controller",
 		Short: "Adds a new controller pkg",
 		Long: `operator-sdk add controller --kind=<kind> --api-version=<group/version> creates a new
```

Comment:
Might want to re-state a simple intro line as the first line of the description and make this paragraph the second paragraph of the description. I'd probably also suggest not leaving the full command syntax here since we have an examples section.

```
Add a new controller package to your operator project.

This command creates a new controller package under pkg/controller/<kind>
...
```

The other thing to check would be the number of columns in a line. To make things pretty on most terminals, we should ensure that no line in the `Long` description goes over 80 characters.

Reply:
I note that if we try to format manually the size it will break the resolution in the terminal.
So, for example, if we have a big one which is one paragraph it will be automatically adjusted to the size of the terminal window. I mean it is responsive.